pkg/joinserver/redis/registry.go

```diff
@@ -121,8 +149,29 @@ func (r *DeviceRegistry) SetByEUI(ctx context.Context, joinEUI types.EUI64, devE
 				return err
 			}
 			f = func(p redis.Pipeliner) error {
-				_, err := ttnredis.SetProto(p, k, stored, 0)
-				return err
+				if _, err := ttnredis.SetProto(p, k, stored, 0); err != nil {
```

Comment:
1. Why change this?
2. The convention is to *not* use short `if` here

Reply:
Why not? Which convention is that?

Reply:
We agreed to use short `if` form only in cases where the there's 1 return value(e.g. `if err :=` ).

Reply:
I don't recall that and it's not in `CONTRIBUTING.md` so we didn't formalize it either.

Check `$ grep -RE 'if ([a-zA-Z]+|_), err :=' pkg` and you'll find 27 other occurrences which are all valid to me.